src/rt/sections_elf_shared.d

```diff
@@ -753,8 +753,15 @@ void scanSegments(in ref dl_phdr_info info, DSO* pdso) nothrow @nogc
 
         case PT_TLS: // TLS segment
             assert(!pdso._tlsSize); // is unique per DSO
-            pdso._tlsMod = info.dlpi_tls_modid;
-            pdso._tlsSize = phdr.p_memsz;
+            version (CRuntime_UClibc)
+            {
+              pdso._tlsSize = phdr.p_memsz;
```

Comment:
Set in both so put it outside. Best way to do this change is to put a `version(uclibc) {} else` on the line before the `_tlsMod` statement you want omitted.

Reply:
done. thanks!

Reply:
I'm not so sure if I agree with this style though.